### PR TITLE
update for craco 7

### PR DESCRIPTION
**Pull request by untra**
Craco 7 is preparing for a major release soon and we would like to see continued support from this plugin:
https://github.com/dilanx/craco/releases/tag/v7.0.0-alpha.8

If theres another way this should be implemented let me know. We look forward to continued use of this plugin.
Thanks for your support. No worries if you want to close this PR and implement changes separately.

### CODE REVIEW

File: packages/craco-less/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@semantic-ui-react/craco-less",
-  "version": "2.0.2",
+  "version": "2.1.0",
```
Comment (layershifter) [inline packages/craco-less/package.json]:
```suggestion
  "version": "3.0.0",
```

Let's bump this to v3 as it's a major bump of core dependency

File: packages/craco-less/package.json [hunk 2]
```diff
@@ -21,13 +21,13 @@
...
   "devDependencies": {
-    "@craco/craco": "^6.4.3",
+    "@craco/craco": "^7.0.0-alpha.8",
     "@pika/pack": "^0.5.0",
     "@pika/plugin-build-node": "^0.8.1",
     "@pika/plugin-standard-pkg": "^0.8.1"
   },
   "peerDependencies": {
-    "@craco/craco": "^6.0.0",
+    "@craco/craco": "^7.0.0",
```
Comment (layershifter) [inline packages/craco-less/package.json:30]:
```suggestion
    "@craco/craco": "^7.0.0-alpha.8",
```

This should be `-alpha` currently as there is no stable currently.

**Comment by layershifter** [comment]
Please also do following:
- run `yarn` in root to update `yarn.lock`

---

P.S. Did you test it with Craco 7, does it actually work? 😃

**Comment by taiio** [comment]
could you please update Craco 7? I have some problem when run npm i with this package. the version Craco 7 my repos conflict with Craco of @semantic-ui-react/craco-less :(

**Comment by untra** [comment]
@layershifter updated with major release version `3.0.0` and dropped the `-alpha` after craco released https://github.com/dilanx/craco/releases/tag/v7.0.0

> P.S. Did you test it with Craco 7, does it actually work?

Testing looks good on our end, but the dependency matrix for craco/semantic/less is pretty big here, and this is a pinch point for meaningfully upgrading dependencies. Can always patch release anything else that needs fixing 👍 

@t-pk and others want to see support, @layershifter can I get another set of 👀 on this?

**Comment by jeffras** [comment]
I'm watching this PR as well, hoping we can get it merged in and clean up our npm warnings. :)

File: packages/craco-less/package.json (hunk 2 above)
Comment (layershifter) [reply packages/craco-less/package.json:30]: v7 was released, not relevant anymore.

**Comment by layershifter** [comment]
@untra @jeffras sorry for being slow there, I missed notifications for this PR 😿  I released `@semantic-ui-react/craco-less@3.0.0`, please check 🚢

**Comment by jeffras** [comment]
You are awesome! Thank you @layershifter

**Comment by JuanSeBestia** [comment]
I get

``` bash
npm WARN ERESOLVE overriding peer dependency
npm WARN While resolving: craco-less@2.0.0
npm WARN Found: @craco/craco@7.1.0
npm WARN node_modules/@craco/craco
npm WARN   dev @craco/craco@"^7.1.0" from the root project
npm WARN   1 more (@semantic-ui-react/craco-less)
npm WARN 
npm WARN Could not resolve dependency:
npm WARN peer @craco/craco@"^6.0.0" from craco-less@2.0.0
npm WARN node_modules/craco-less
npm WARN   craco-less@"^2.0.0" from @semantic-ui-react/craco-less@3.0.0
npm WARN   node_modules/@semantic-ui-react/craco-less
npm WARN 
npm WARN Conflicting peer dependency: @craco/craco@6.4.5
npm WARN node_modules/@craco/craco
npm WARN   peer @craco/craco@"^6.0.0" from craco-less@2.0.0
npm WARN   node_modules/craco-less
npm WARN     craco-less@"^2.0.0" from @semantic-ui-react/craco-less@3.0.0
```

So, could you update to version `craco-less@2.0.0` to `craco-less@^3.0.1`, that it support `craco@6 | craco@7` in the packages ?
https://github.com/layershifter/semantic-ui-tools/blob/master/packages/craco-less/package.json#L21

**Pull request merged**